tla-bmcmt/src/test/scala/at/forsyte/apalache/tla/bmcmt/TestArena.scala

```diff
@@ -30,7 +30,7 @@ trait TestArena extends FixtureAnyFunSuite {
 
   test("BOOLEAN has FALSE and TRUE") { _ =>
     val arena = Arena.create(solver)
-    val boolean = arena.cellBooleanSet()
+    arena.cellBooleanSet()
```

Comment:
this looks like a fruitless call, since `cellBooleanSet` should not have side effects